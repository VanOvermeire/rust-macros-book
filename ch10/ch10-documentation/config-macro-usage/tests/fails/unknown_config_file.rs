use config_macro::config;

config!(path = "./fake.yaml");

fn main() {}