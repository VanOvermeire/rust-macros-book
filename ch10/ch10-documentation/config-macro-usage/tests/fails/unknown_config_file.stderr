error: could not find config with path ./fake.yaml
 --> tests/fails/unknown_config_file.rs:3:1
  |
3 | config!(path = "./fake.yaml");
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `config` (in Nightly builds, run with -Z macro-backtrace for more info)
